lib/ohai/plugins/freebsd/network.rb

```diff
@@ -68,6 +68,7 @@
         end
       end
       if line =~ /\s+inet6 ([a-f0-9\:]+)%?(\w*)\s+prefixlen\s+(\d+)\s*\w*\s*([\da-fx]*)/
+        next if $1 == "::1"
```

Comment:
I'm not sure if this is the right approach. Is `::1` the only `inet6` line in your `ifconfig`?

Reply:
No, I have fe80 addresses on all the interfaces and those are properly excluded.

Sometimes ::1 is on lo1 and not lo0, but it should always see my inet6 addresses starting with 2600 first and it doesn't pick them up. It always takes the ::1 as the default address for some strange reason.

On a very vanilla out-of-the-box FreeBSD install with IPv6 enabled Chef will pick up ::1 as the IPv6 address. This is the only way I found to fix it.